src/quo/components/tabs/tabs/schema.cljs

```diff
@@ -0,0 +1,32 @@
+(ns quo.components.tabs.tabs.schema)
+
+(def ^:private ?data-schema
+  [:sequential
+   [:maybe
+    [:map
+     [:id [:maybe :keyword]]
+     [:label [:maybe :string]]
+     [:accessibility-label {:optional true} [:maybe [:or :keyword :string]]]
+     [:notification-dot? {:optional true} [:maybe :boolean]]]]])
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map
+      [:default-active {:optional true} [:maybe :keyword]]
```

Comment:
When opening the AC, the schema fails because we pass `:default-active` as a number, not as a keyword.